src/main/java/org/folio/roles/mapper/KeycloakPolicyMapper.java

```diff
@@ -263,7 +263,7 @@ protected static PolicyLogicType parsePolicyLogic(PolicyRepresentation policyRep
     return ofNullable(policyRepresentation)
       .map(AbstractPolicyRepresentation::getLogic)
       .map(Enum::name)
-      .map(enumValue -> PolicyLogicType.fromValue(enumValue.toLowerCase()))
+      .map(enumValue -> PolicyLogicType.fromValue(enumValue.toUpperCase()))
```

Comment:
this should be fixed, in my test
![image](https://github.com/user-attachments/assets/7d48380b-98ec-47be-b87b-5ae147542e52)